### ISSUE TITLE
Poor communication leads to many problems

### ISSUE THREAD

**Issue by TzviyaSiegman** [w3c/w3process#690]
Poor communication in the community leads to multiple problems:
Summarizing some problems, not yet sure of proposed solutions (see also #620 and #328)
* Few AC reps respond to the majority of charter and spec reviews
* Current process has a bias in favor of approving charters and specs (no need for an explanation when approving, need to support objections) (this isn't necessarily a problem, but it is a bias)
* It is easier to formally object to a charter/spec than it is to comment in earlier stages of a proposal. This leads to problems at the later stages of charter/spec and fosters resentment at those filing legitimate concerns because of the lateness.

Some areas to explore:
* Consider our expectations of AC reps. AC reps get dozens of emails a week and are expected to vote on hundreds of issues a year. Is this appropriate? What can be improved?
* Communication is limited to within each WG and within the AC. How can we communicate with the larger community?
* How can people be informed of work effectively at an earlier stage so that we avoid FOs and get input?

**Comment by cwilso** [comment]
Although I agree with your basic premise (in the title), this... unfortunately seems like it belongs to the AB at this point, not the Process?

**Comment by dwsinger** [comment]
As an AC Rep I found it easy to keep up with the voting requirements. Every friday I checked "My Questionnaires". W3C+INCITS+CTA took me on average 20-30 mins a week

**Comment by tobie** [comment]
> As an AC Rep I found it easy to keep up with the voting requirements. Every friday I checked "My Questionnaires". W3C+INCITS+CTA took me on average 20-30 mins a week

Clearly others don't. Myself included. I personally would find it a lot easier to get involved if we were able to have a public discussion about the topic (like in an issue here, for example), rather than just have to vote by myself. I don't necessarily always have the time, the context, or the expertise to be able to assess a charter. But I'd certainly learn a lot if there were was a public (or member-only) discussion about each one

**Comment by dwsinger** [comment]
Sorry Tobie, you are right. Finding comments and discussion about open polls is quite hard, I agree. I agree we can and should improve discussion. I was reacting to the "voting is per se a burden", which I don't agree with.

Perhaps every poll (not election) should have a 'primary' GH issue in which discussion is anchored (and separate issues could be opened if it gets complex)?

**Comment by tobie** [comment]
> Perhaps every poll (not election) should have a 'primary' GH issue in which discussion is anchored (and separate issues could be opened if it gets complex)?

That seems like an easy and well-scoped enough idea to experiment with. I like it.

**Comment by michaelchampion** [comment]
> AC reps get dozens of emails a week and are expected to vote on hundreds of issues a year. Is this appropriate? What can be improved?

Hmm, I suspect many of those same AC reps see and possibly react to 10x or 100x as many social media posts, DMs and text messages, etc. Not to mention work emails ...  Information overload and selective attention / filter bubbles is a general problem (exacerbated by the Web). 

Certainly an interesting topic to explore.  Possibly one of those things our many members in academia can help with? Not sure it's a useful Process issue until there a rough idea of how it could be addressed.

> Current process has a bias in favor of approving charters and specs (no need for an explanation when approving, need to support objections) (this isn't necessarily a problem, but it is a bias)

This IS a Process issue, but probably separate from the Communications question.

**Comment by michaelchampion** [comment]
> Communication is limited to within each WG and within the AC. How can we communicate with the larger community?

My initial thought is to be massively more transparent, restricting Member Confidential information to only the most truly sensitive matters.  That would allow the search engines to index, AI bots to learn from, and bloggers to analyze and summarize W3C information.

 OK, that would also inflict  the bad things the web has enabled on W3C's workings (trolls, data brokers, scammers, toxic political narratives, ad nauseam)  ... but maybe making us eat that toxic stew within W3C will motivate us to take stewardship of the web to help clean up the messes it enabled 🤔

**Comment by caribouW3** [comment]
> > Perhaps every poll (not election) should have a 'primary' GH issue in which discussion is anchored (and separate issues could be opened if it gets complex)?
> 
> That seems like an easy and well-scoped enough idea to experiment with. I like it.

The Process already has Advance Notices for charters to be discussed before the vote. The place for discussion is generally a public mailing list. Are you suggesting that we just move that to a centralized place like a single GH repo for all the charters?
It would probably not get all the material/discussions that happen before the sending of the advance notice, though, so I'm not sure it'd really give more insight to the AC.

**Comment by tobie** [comment]
> Are you suggesting that we just move that to a centralized place like a single GH repo for all the charters?

Yes, or possibly even in a GitHub repository for the whole AC? Thankfully we can move issues around now, so we could try out something and eventually move issues around if needed.

**Comment by michaelchampion** [comment]
As I understand the proposal, there would be a GitHub issue or document to *anchor* discussion of a particular charter proposal, from brainstorming, through Advance Notice, and up through AC Ballot..  That doesn't mean discussion couldn't happen on a mailing list (or Slack channel, in minuted meetings, whatever), just that someone coming to the question can find where to pick up the threads and follow them.

**Comment by fantasai** [comment]
For chartering, we had suggested that the charter be proposed to the AC together with a Disposition of Comments ([sample](https://drafts.csswg.org/css-flexbox-1/issues-cr-2012)), so that the comments on the charter prior to AC review (and how they were addressed) could be made visible. Anchoring that to a new thread once the AC Review kicks off might be a way to have all the relevant information available (and summarized!) while not forcing the entire charter drafting conversation into a single GH thread (which is totally unmanageable).

As for organizing votes and stuff, it would be useful if such a participant-specific dashboard was one or two obvious clicks from the w3.org home page. (I've never quite figured out how to navigate the w3.org home page.)

**Comment by tobie** [comment]
> (I've never quite figured out how to navigate the w3.org home page.)

Yeah, the cognitive overload of navigating the website and the auth system in particular is very real. I'm not going to lie; I struggle.

**Comment by cklim483** [comment]
> Poor communication in the community leads to multiple problems:
> Summarizing some problems, not yet sure of proposed solutions (see also #620 and #328)
> * Few AC reps respond to the majority of charter and spec reviews
> * Current process has a bias in favor of approving charters and specs (no need for an explanation when approving, need to support objections) (this isn't necessarily a problem, but it is a bias)
> * It is easier to formally object to a charter/spec than it is to comment in earlier stages of a proposal. This leads to problems at the later stages of charter/spec and fosters resentment at those filing legitimate concerns because of the lateness.
> 
> Some areas to explore:
> * Consider our expectations of AC reps. AC reps get dozens of emails a week and are expected to vote on hundreds of issues a year. Is this appropriate? What can be improved?
> * Communication is limited to within each WG and within the AC. How can we communicate with the larger community?
> * How can people be informed of work effectively at an earlier stage so that we avoid FOs and get input?
>

**Comment by TzviyaSiegman** [comment]
@cwilso wrote

> Although I agree with your basic premise (in the title), this... unfortunately seems like it belongs to the AB at this point, not the Process?

I'll open a parallel issue in the AB repo after this gets fleshed out a little more. I agree that the basic communications issue is an AB thing, not a Process thing.

**Comment by TzviyaSiegman** [comment]
> As for organizing votes and stuff, it would be useful if such a participant-specific dashboard was one or two obvious clicks from the w3.org home page. (I've never quite figured out how to navigate the w3.org home page.)

I have not figured out how to navigate this either. I have missed votes because I have not figured out a system like the one @dwsinger describes to conveniently organize the votes I need to get to in a given week. My email system is not good at organizing or searching, so I don't have the luxury of relying on that. I think we will see a big bump in voting if we provide people with ways to access voting information other than just an email.

**Comment by nigelmegitt** [comment]
> * Current process has a bias in favor of approving charters and specs (no need for an explanation when approving, need to support objections) (this isn't necessarily a problem, but it is a bias)

@TzviyaSiegman you list this amongst the problems; I think it's an entirely appropriate bias and changing it _would_ be a problem.

Re the ability to comment and have access to draft Charters in a GH repo, as a data point, this is exactly what we've done with the [TTWG Charter Draft repo](https://github.com/w3c/charter-timed-text/) for many years. Whether it's done in separate repos or one big one, I don't think really matters, though single document repos are generally better for the simplicity of seeing issues and pull requests.

Certainly a repo provides a better space for commenting on a document than an email reflector.

Putting both these thoughts together, keen observers will have noticed that the TTWG Charter received FOs at AC review time, one of which was from a WG member, Apple, who had a lot of opportunities over a very long period of time to comment at the charter drafting stage, on that repo, or in WG meetings, but did not do so until AC review. That possibly points to another problem, to do with prioritisation and time availability, and I don't know how we can address that.

Observation: participating in ahead-of-time discussions can cost more for the individual but speeds up the process for the group; conversely waiting can mean the individual spends less time at the cost of imposing late and unbounded delays for the group. If anyone knows of a model that increases efficiency for everyone, let's hear it!

**Comment by tobie** [comment]
> If anyone knows of a model that increases efficiency for everyone, let's hear it!

There's a lot to learn about the sales process which has honed its ability to lower acquisition costs over millenaries. In sales, a lot is done to address objections upfront. This implies proactively imagining and collecting objections beforehand, and sometimes modifying what it is that you are selling accordingly.

My suggestion here would be to normalize upfront outreach to the various constituencies for whom the scope of the charter is relevant. This helps understand upfront what their objections or concerns will be and allows modifying the charter accordingly (when that's possible) or address those issues upfront, maybe even in an accompanying document (like an FAQ, for example).

I'm sure a lot of that is already done somewhat organically, but that process could be formalized a little the same way the WICG formalized spec explainers for instance.

Maybe creating a default repository with a checklist, FAQ, explainer, etc., for charter creation would be helpful.

**Comment by dwsinger** [comment]
w3.org -> Membership -> Member Home -> My … Reviews, votes and other questionnaires

You calendar is there in User Account

**Comment by TzviyaSiegman** [comment]
@dwsinger I believe that you have just unlocked a secret for many people, myself included. I had no idea that was there. I had never before notices the link "member home". We can discuss with those working on the website redesign how buried that is.

**Comment by michaelchampion** [comment]
@tobie wrote
> My suggestion here would be to normalize upfront outreach to the various constituencies for whom the scope of the charter is relevant. This helps understand upfront what their objections or concerns will be and allows modifying the charter accordingly

+1.  That addresses the "bias toward charter approval" we've been discussing.  IMHO the burden of proof should be on those who think W3C should invest its scarce resources, and the web community invest its scarce attention, on a new standards effort.  Charter proponents should be seeking out  those most likely to have concerns and working to resolve them up front.  And for that matter, the AB or the Process CG should be looking for patterns in objections that might be addressed by clearer Process language about things like interoperability, now there is no Director to make nuanced judgments.

"You had this information somewhere in one of the inboxes of one of the busy people who track a community as one small part of their job, but didn't object until the charter went to the AC" is a reasonable pushback IN PRINCIPLE, but not  in the reality of W3C's tangled communications systems.

**Comment by cwilso** [comment]
I don't think it is appropriate to try to focus on outreach to those for whom "the scope of the charter is relevant" - both because we should be aiming for a more principled W3C that chooses carefully what it invests its efforts in, and because we should be ensuring that we are consistent across all work with the quality and investment bar we expect.

For example, to use @nigelmegitt's TTWG charter example, Apple's FO was about the quality bar of how we require multiple implementations, not about Timed Text itself [*].  Although it's true they could have made this comment at any time, I don't know that it would have been fixed early in the process either; but it's not entirely appropriate to expect them to have participated heavily in editing a recharter of a WG that they don't participate in.  This was a lowering of a bar that Apple (and others) care about quite a bit.

As to @michaelchampion's comment, it ISN'T rational to expect that those who consider themselves some kind of stewards of the community will be able to watch everything, all the time - we need to make better inflection points (and possibly more of them), and we need to make 

As part of the Incubation priority that the AB has on their plate ([link](https://www.w3.org/wiki/AB/2023_Priorities#The_4_Is:_Interoperable.2C_independent.2C_implementation.2C_Incubation)), but we haven't really seriously worked on yet, I envision that we need to have a consistent pipeline for managing work, from incubation to chartering WGs to delivering RECs, and we need to have a clearer map of where each bit of work is on this pipeline.  The CSS WG's "Current Work" page ([link](https://www.w3.org/Style/CSS/current-work.en.html)) is a reasonable example of this, though I think each line item would need to have more information; and we need to have a clearer notification system for transitioning through these stages.  (This is not entirely unlike the system we have built for [Chromestatus](https://chromestatus.com/) and the Chromium launch process.)

For example, I would expect that if we have a clearer way to hear the message "hey, we've drafted our WG recharter, PTAL before it goes to a vote", along with better diffs for what has changed and what changes between that phase and the AC vote, we might be able to work this more smoothly.  (I myself am apparently currently watch 290 repos - so, noise.) Having a broader, open "review group" that took responsibility for giving these things the once-over might help, as well, particularly if we could require those who frequently FO to participate.

[*] please note this is a shorthand for my interpretation of this FO; it is not intended to function as a statement of any kind, merely illustrative.

**Comment by chaals** [comment]
It seems this issue is not about the general fact that poor communication causes problems but about specific issues related to chartering. @TzviyaSiegman would you please name it something more closely tied to the specific issues you're hoping to address?

**Comment by michaelchampion** [comment]
Well, the chartering issues are presumably a symptom of the more general communications problem. @TzviyaSiegman 's original point
> It is easier to formally object to a charter/spec than it is to comment in earlier stages of a proposal. This leads to problems at the later stages of charter/spec and fosters resentment at those filing legitimate concerns because of the lateness.

Gets to the heart of the communications problem. As does @cwilso's comment about 

> I envision that we need to have a consistent pipeline for managing work, from incubation to chartering WGs to delivering RECs, and we need to have a clearer map of where each bit of work is on this pipeline. 

Perhaps Tzvia's point
> Current process has a bias in favor of approving charters and specs (no need for an explanation when approving, need to support objections)

Is a separate issue.  But it has a communications aspect to it:  In the absence of broad feedback, does W3C default to giving something its imprimatur, or does the process block until there is affirmation that some group is on the right course?

**Comment by Fb9027** [comment]
The starter, please

**Comment by Fb9027** [comment]
- - [ ] @

**Comment by dwsinger** [comment]
It's true Apple is a member of the TTWG.

But it's not the bias to approval so much as the risk there being no-one replacing the Director's role in checking "should we really be doing this?". As I say, something that is anybody's easily becomes nobody's job. I think we have a significant risk of matters slipping by the rubber stamp and (if we are lucky) causing more grief later in the pipeline.

**Comment by michaelchampion** [comment]
> The current bias towards Charter approval is slender, but it's the right bias.

@fantasai @frivoal @cwilso @TzviyaSiegman  (Process editors and AB Chairs) please advise:  This seems to be an issue in a number of threads, should it be addressed in the Process or the Vision https://github.com/w3c/AB-public/issues?

I can imagine that it's a Process issue in that https://www.w3.org/2021/Process-20211102/#WGCharterDevelopment and https://www.w3.org/2021/Process-20211102/#implementation-experience assume there is an engaged Director who is making informed, principled judgments about the suitability of charters and the evidence that a spec is mature enough to be a Recommendation.  Without a Director, perhaps the Process should give more explicit guidance.

I can also imagine this is a Vision question:  Is the purpose of W3C to facilitate groups getting together to produce clearly structured and well-reviewed *documents* that do not get sustained objections, or is it to produce *web technology specifications* that provably advance the values (interoperability, accessibility, internationalization, privacy, etc.) of the  organization?  If the point of W3C is to produce "documents" I'd agree with @nigelmegitt that a bias toward charter approval (with relatively lenient success criteria) is appropriate.  If the point of W3C is to produce well-vetted web technology specs, requirements should be rigorous at each stage. 

While I prefer a high incubation bar to create a WG and a high implementation / interoperability bar to get to Recommendation, this has created tensions (when I was on the AC and AB) because many seems to envision W3C as more of an environment to encourage  specs that MIGHT be useful than a place to filter out ideas that do not (yet) interoperate in the real world or cause toxic side effects on W3C's other values.

**Comment by cwilso** [comment]
@michaelchampion : I think this is actually squarely a question at the heart of the I3 (Incubation, Interop, Implementation) priority.  It probably feeds into the Vision at some point, but we need to get some concepts on the process of incubation sorted first.

@nigelmegitt I would call out that there are a few things at play here, and as one of the relatively few "really big companies that do a lot of things at W3C" AC reps, I wanted to explain how things fall through the cracks sometimes.  There's a big disconnect sometimes at large companies between the engineers on the ground who work on particular efforts and the "centralized standards" team. (That's in quotes, as it's rarely actually centralized, per say, and it's frequently not a defined, reporting-hierarchy team.)  

For example, Google has one person who is a participant in TTWG.  I've never met them in person that I recall; their participation makes sense, I've reviewed TTWG's charter in the past, and it's worthwhile to Google to support this effort with their IP.  At the same time, that person is not deeply ingrained in the "how should standards be done" discussions/mindset/ethos that a number of us who do standards as our primary job are.  It can be a lot more difficult for me, for example, to keep on top of all the charters under revision ahead of schedule; but I care quite deeply about, say, a convenient change to requirements on a charter that actually translates to "multiple implementation interoperability is no longer a baseline requirement".  The Google participant in TTWG probably doesn't know that or understand, probably hasn't looked at that part of the charter, and trusts that the Chairs and the process are just "taking care of this paperwork so we can keep working on this important bit of API surface".

(Apple has more senior people, including an AC rep, as participants of the TTWG; at the same time, I would say I'm joined to some groups I don't really participate in or even pay much attention to.)

I don't have an answer to how to do this better to avoid this sort of thing, other than as I've suggested recently having a "review team" that has dedicated bandwidth from "the usual suspects" (e.g., ESPECIALLY anyone who has FOed to charters recently for this kind of process grounds) to do a pre-review before this goes to an AC vote.   Probably this would have to be like "Charter/REC pre-review CG".  It wouldn't be binding, but it might help pre-identify these kinds of issues and have an open discussion about them   The hardest part of this would be 1) getting someone to consistently lead it, though perhaps this fits in with Team duties, and 2) ensuring that we get the low-latency dedicated bandwidth from the usual suspects.  But from my perspective, it would just move the detailed review I have to do of charters to an earlier point, and would be worth the additional bandwidth to make this more efficient and friendly.

**Comment by michaelchampion** [comment]
Thanks @cwilso .  On reflection, I agree this is not a "Vision" issue at this point because W3C's Vision should encompass exploration, experimentation, innovation etc. (via IGs, CGs, Workshops, etc.) as well as a rigorous Rec Track process that requires successful incubation (and values review) before chartering and enforces stringent success criteria on all charters. But where can the larger community observe and comment on actual work towards the AB's I3 priority (ahem, https://www.w3.org/wiki/AB/2023_Priorities calls it "I4" ;-) ) ??

**Comment by mnot** [comment]
Back to the original issue...

I asked for a community-wide mailing list at one of the monthly AC sessions a while back, and it seemed like there was good support for it at the time. What happened with that?

I can't get excited about a github project/issues list here, because we need a community _dialogue_ / _public square_. An issues list empowers whoever controls it to close issues and generally shape the discussion. And for most people, it's still a dashboard that they have to check in on (yes, you can subscribe to GH issues, but most don't).

Of course, that doesn't preclude gh issues/projects being used to organise thoughts and allow engagement by people who work best in that mode. They're just not sufficient on their own.

**Comment by tobie** [comment]
@mnot [wrote](#issuecomment-1399837266):
> I asked for a community-wide mailing list at one of the monthly AC sessions a while back, and it seemed like there was good support for it at the time. What happened with that?

PWE [raised concerns](https://www.w3.org/2022/12/20-pwe-minutes.html) around the inclusivity of the AC Forum mailing list recently. I think we'd need to solve that problem before extending a similarly-themed mailing list to a broader audience.

**Comment by cwilso** [comment]
FWIW, we DO have the W3C Community Slack.

**Comment by michaelchampion** [comment]
@mnot what's an example of an open community-wide mailing list that W3C might use as an example.  The IETF "general discussion" list archived at https://mailarchive.ietf.org/arch/browse/ietf/ maybe?

**Comment by mnot** [comment]
The IETF list is an example, and also a warning. That community has some issues that play out poorly on the list. I suspect a W3C effort would be much better, because community standards are more established here, more widely subscribed to, and backed up with various mechanisms. Still, it's good to remember we'll need to think about moderation, de-escalation, etc. A list on its own won't do any good; we need to make sure that discussion is healthy, and that the community's efforts get channeled into the appropriate places off the list.

@tobie I think the issues raised at PWE are _because_ of the fragmented, non-inclusive and high-stakes nature of the AC list. Representing a company instead of just yourself makes things more fraught; not allowing community members to have their personal say becuase they're not the official representative is damaging. I don't think we should gate other improvements on the successful rescue of that venue.

**Comment by TzviyaSiegman** [comment]
I don't necessarily agree that the issues are caused by the high-stakes nature of ac-forum, but I don't think we need to wait until we fix those problems because we might have to wait for a very long time. We can learn from the problems though. We know that we need to establish a list moderator - not someone who approves every message but someone(s) who are responsible for stopping unacceptable behaviors and encouraging good behaviors. These people can help improve the tone of the conversation and put an end to discussions that are damaging. We (possibly PWE) will have to help with some guidance. Let's focus on the logistics of how to improve community-wide communication through something like an email list. Should it be opt-in or opt-out?

**Comment by dwsinger** [comment]
I think there is a pro-active aspect to this also; "we, members, need to be talking about X". Email lists are good for discussion, but maybe the AC also needs its issue list and Github repo? That's a better way to track discussion on a specific question, I think (as we see here)

**Comment by frivoal** [comment]
This is an good discussion, but it's not clear to me that it belongs in the ProcessCG. I'd suggest tranfering this to https://github.com/w3c/AB-memberonly/ or https://github.com/w3c/AB-public

cc: @cwilso @TzviyaSiegman as AB chairs to decide where to put this.

**Comment by michaelchampion** [comment]
Some personal takeaways after re-reading this thread (before it moves):
- I'm persuaded that we need some sort of community-wide (perhaps member-only, but NOT AC-only) forum to share ideas on what's happening with the Web and what W3C can do to make it better. I hope @mnot is right that W3C's culture will prevent the sort of toxic months-long threads IETF had on code-of-conduct issues.
- While all sorts of mechanisms feel comfortable for different people, adopt @dwsinger 's suggestion to anchor topics in GitHub so people can (hopefully) find pointers to the repos, mailing lists, ballots, Slack channels, IRC logs,  Mastodon threads, etc. where interesting discussions happen.
- Updating pointers about what's happening and where the the discussions are taking place will require some sort of moderator(s).  And as @TzviyaSiegman notes, moderation is needed to keep the discussions constructive, discourage bike shedding,  privately call out people who make others uncomfortable, and try to shut down discussions that are not going anywhere. And closing the "anchor" issues when appropriate, probably after some sort of call for consensus or straw poll.
- I know some don't think any of the  overworked AB/TAG/Team/chair/Director folks will step up to moderate such a forum.  I'm a bit more optimistic; I suspect there are a number of people who would find personal satisfaction in helping make W3C discussions more useful and constructive, less intimidating, etc.

**Comment by TallTed** [comment]
> FWIW, we DO have the W3C Community Slack.

YMMV, but I generally find Slack to be yet another tragic attempt to reimplement NNTP, which doesn't handle threads helpfully, doesn't foster my workflows, and generally leaves me wanting more and different.  It certainly doesn't replace SMTP/POP3/IMAP4 personal and distro email, which itself of course remains imperfect, but has the weight of decades behind it.

**Comment by tobie** [comment]
Agree with @TallTed re Slack. Its threading is awful. It has all of the issues of a mailing list with none of its benefits. It's at most a possible contender to IRC in our toolbox, not to mailing lists.

Thanks for the great summary, @michaelchampion. A few things:

* There's great value in broadening the conversation to all of the membership, or even to the general public, but it is important for us to note that not everyone is confortable operating in more open/public circles (for a number of different reasons). And while it is normal to expect transparency from elected bodies, we should carefully consider its impact on inclusivity for the ac-forum or a broadened version of the ac-forum.
* Do we have a sense of the impact that broadening those discussions forums would have on membership dues? Very candidly, I pay membership fees for my consulting firm essentially to have access to the ac-forum, and probably would stop paying them if I could get access regardless. _(I feel like a contribute a fair share already by paying for the hosting of Specref and PR Preview.)_
* Regardless of wether a number of folks step up to help with moderation, mentoring, etc., the ac-forum needs a chair (or maybe even co-chairs), and even more so if the group gets extended.

**Comment by dwsinger** [comment]
Slack is also a single-vendor proprietary system and I'd prefer not to use such if we can possibly avoid it. (And I agree, the main advantage I see in Slack is the lack of junk and having to check the filters every day, which isn't much.)

**Comment by cwilso** [comment]
So many things to respond to!  :)

1. @frivoal I agree this belongs in AB-space (See my first comment.). I don't have admin rights in /w3process to transfer it (and it should definitely be transferred, not copied or a new issue created).  If you or someone with rights wants to transfer it, go for it, or give me admin rights here.  I think it should go to memberonly.
2. I was not recommending Slack as a tool, I was pointing out we ALREADY HAVE a Member-wide discussion forum.  Many of you will have heard my anti-Slack rants before.  :)
3. I would point out that the stewardship/chairing issues will be even harder with a Member-wide list than an AC forum list, just because there are more people.  (E.g., you will have 650 Googlers possibly participating, rather than two.)
4. To @tobie's point, I would recommend a broader discussion forum should include Member (and Invited Expert) participants only, not the broader public, and still be Member-confidential.  The engagement you will get from me on public vs Member-confidential fora will differ significantly, e.g.

**Comment by tobie** [comment]
> I was not recommending Slack as a tool

I didn't mean to imply you were. That was a knee-jerk reaction. Apologies.

**Comment by cwilso** [comment]
@tobie not at all; it wasn't just you.  (FWIW, Slack threading is a "you get out of it what ALL of you put into it" proposition - as @tantek will tell you, if you cajole everyone to properly thread things, it's... better.  Still not ideal, of course.
